kotsadm/operator/pkg/client/deploy.go

```diff
@@ -514,6 +514,30 @@ func (c *Client) installHelm(helmDir string, namespace string) (*commandResult,
 	return result, nil
 }
 
+func (c *Client) uninstallWithHelm(helmDir string, namespace string, charts []string) error {
+	version := "3.4.2"
+	if os.Getenv("KOTSADM_TARGET_NAMESPACE") != "" {
+		namespace = os.Getenv("KOTSADM_TARGET_NAMESPACE")
```

Comment:
Don't you have to use the namespace in the kots.io hemlchart if set?

Reply:
Good catch.  Updated to use the same logic as `kubectl apply` commands.